static/src/javascripts/projects/common/modules/commercial/creatives/template-preprocessor.js

```diff
@@ -7,6 +7,7 @@ define([
     'text!common/views/commercial/creatives/manual-multiple-button.html',
     'text!common/views/commercial/creatives/manual-title.html',
     'text!common/views/commercial/creatives/gimbap/gimbap-simple-blob.html',
+    'text!common/views/commercial/creatives/gimbap/gimbap-richmedia-blob.html',
```

Comment:
This module is getting quite large, I'll add this to the list of things to improve during my tenure in the health team 🎅